packages/app/src/LogSidePanel.tsx

```diff
@@ -1978,6 +1980,8 @@ function SidePanelHeader({
     parsedProperties?.['process.tag.rum.sessionId'] ??
     sessionId;
 
+  const playerExpanded = useAtomValue(playerExpandedAtom);
```

Comment:
mega nit: I think it'd make more sense if we named this as like "collapseEventContext" or something right? I can imagine technically any subpanel might want in the future to be able to collapse the event context around it. Though not terribly familiar with jotai and it seems like the data flow is somewhat inverted to what I'm used to.